request.js

```diff
@@ -1229,10 +1234,10 @@ Request.prototype.aws = function (opts, now) {
     self._aws = opts
     return self
   }
-  
+
   if (opts.sign_version == 4 || opts.sign_version == '4') {
     var aws4 = require('aws4')
-    // use aws4  
```

Comment:
White space removal? Happened automatically from using my editor. The whitespace was obviously an accident to begin with. If the maintainers want it removed, I'm happy to.